frontend/src/app/modules/common/resizer/resizer.component.ts

```diff
@@ -1,22 +1,29 @@
-import {
-  Component,
-  OnDestroy,
-  EventEmitter,
-  Output,
-  Input,
-  HostListener} from "@angular/core";
+import {Component, EventEmitter, HostListener, Input, OnDestroy, Output} from "@angular/core";
 
 
 export interface ResizeDelta {
-  x:number;
-  y:number;
+  origin:MouseEvent;
+
+  // Absolute difference from start
+  absolute:{
+    x:number;
+    y:number;
+  };
+
+  // Relative difference from last position
+  relative:{
+    x:number;
+    y:number;
+  };
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/d356c82449c2301a333e72b8f2c7e05463706d4a/frontend/src/app/modules/common/resizer/resizer.component.ts#L19"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>